core/person.rb

```diff
@@ -1,8 +1,8 @@
-require_relative 'nameable'
+require './switches/nameable'
 
 class Person < Nameable
   attr_reader :id
-  attr_accessor :name, :age, :books, :rentals
+  attr_accessor :name, :age, :books, :rentals, :parent_permission
 
   def initialize(age, name = 'Unknown', parent_permission: true)
```

Comment:
- [ ] I appreciate your effort in solving different bugs 👏.  However, there are still some errors that are preventing your app from running.

![image](https://user-images.githubusercontent.com/90222110/187494631-a4ceb69e-f5c5-4211-8d9a-2410e89e98c1.png)

I believe that you can resolve this error also, so please resolve it and make sure that everything is fine 😉